Description: Tweak sane-desc to suit our needs
 Do not link to HTML manpages, we do not have them installed. Also,
 do not use sane.png, as it's not distributed in the source tarball.
 Do not output a timestamp as part of the udev rules file header, as
 this breaks multiarch.
Author: Julien BLACHE <jblache@debian.org>

Index: sane-backends-1.0.22/tools/sane-desc.c
===================================================================
--- sane-backends-1.0.22.orig/tools/sane-desc.c	2011-06-13 15:35:40.875152727 +0200
+++ sane-backends-1.0.22/tools/sane-desc.c	2011-11-21 15:51:25.135951802 +0100
@@ -46,7 +46,7 @@
 
 #define SANE_DESC_VERSION "3.5"
 
-#define MAN_PAGE_LINK "http://www.sane-project.org/man/%s.5.html"
+#define MAN_PAGE_LINK "/usr/share/doc/libsane/html/%s.5.html"
 #define COLOR_MINIMAL      "\"#B00000\""
 #define COLOR_BASIC        "\"#FF9000\""
 #define COLOR_GOOD         "\"#90B000\""
@@ -2496,8 +2496,8 @@
 		  printf ("<br>\n");
 		}
 	      if (be->manpage)
-		printf ("<b>Manual page:</b> <a href=\"" MAN_PAGE_LINK
-			"\">%s</a><br>\n", be->manpage, be->manpage);
+		printf ("<b>Manual page:</b>"
+			"%s<br>\n", be->manpage);
 
 	      if (be->comment)
 		printf ("<b>Comment:</b> %s<br>\n", be->comment);
@@ -2731,9 +2731,9 @@
 
 	  printf ("</td>\n");
 	  if (model_record->be->manpage)
-	    printf ("<td align=center><a href=\""
-		    MAN_PAGE_LINK "\">%s</a></td>\n",
-		    model_record->be->manpage, model_record->be->manpage);
+	    printf ("<td align=center>"
+		    "%s</td>\n",
+		    model_record->be->manpage);
 	  else
 	    printf ("<td align=center>?</td>\n");
 
@@ -2758,8 +2758,8 @@
   printf
     ("</head>\n"
      "<body bgcolor=FFFFFF>\n"
-     "<div align=center>\n"
-     "<img src=\"http://www.sane-project.org/images/sane.png\" alt=\"SANE\">\n");
+     "<div align=center>\n");
+  /*     "<img src=\"http://www.sane-project.org/images/sane.png\" alt=\"SANE\">\n"); */
   printf ("<h1>%s</h1>\n", title);
   printf ("</div>\n" "<hr>\n");
   printf ("%s\n", intro);
@@ -3436,8 +3436,8 @@
 {
   time_t current_time = time (0);
   printf ("# This file was automatically created based on description files (*.desc)\n"
-	  "# by sane-desc %s from %s on %s",
-	  SANE_DESC_VERSION, PACKAGE_STRING, asctime (localtime (&current_time)));
+	  "# by sane-desc %s from %s\n",
+	  SANE_DESC_VERSION, PACKAGE_STRING);
 
   printf 
     ("#\n"
